src/libraries/Microsoft.PowerFx.LanguageServerProtocol/LanguageServer/LanguageServer.cs

```diff
@@ -475,6 +479,24 @@ private void PublishTokens(string documentUri, CheckResult result)
                 }));
         }
 
+        private void PublishExpressionType(string documentUri, CheckResult result)
+        {
+            var uri = new Uri(documentUri);
+            var nameValueCollection = HttpUtility.ParseQueryString(uri.Query);
+            if (!bool.TryParse(nameValueCollection.Get("getExpressionType"), out var enabled) || !enabled)
+            {
+                return;
+            }
```

Comment:
Do we need this check? This is only in response to a specific message request, so could "old" clients ever hit this?